### PR TITLE
Helm dashboard moshe

### PR DESCRIPTION
**Pull request by MosheOgalbo**
## Check List

- [ ] The title of my pull request is a short description of the changes
- [ ] This PR relates to some issue: 
- [ ] I have documented the changes made (if applicable)
- [ ] I have covered the changes with unit tests

### CODE REVIEW

File: dashboard/src/components/ShutDownButton.tsx [hunk 1]
```diff
@@ -3,17 +3,21 @@ import { useState } from "react";
 import Modal from "./modal/Modal";
 import axios from "axios";
 
-function ShutDownButton() {
+
+ function ShutDownButton () {
   const onClose = () => {
-    console.log("close button clicked");
+    console.clear();
+    console.log("close button clicked");
```
Comment (hdm23061993) [inline dashboard/src/components/ShutDownButton.tsx:10]: Not needed. Please remove them

File: dashboard/src/components/ShutDownButton.tsx [hunk 2]
```diff
@@ -3,17 +3,21 @@ import { useState } from "react";
...
+    console.clear();
+    console.log("close button clicked"); 
+    setSignOut(false);
   };
   const [signOut, setSignOut] = useState(false);
-
   const handleClick = async () => {
-    await axios
+    console.clear();
+    console.log("open message click ");
```
Comment (hdm23061993) [inline dashboard/src/components/ShutDownButton.tsx:16]: Not needed. Please remove them

File: dashboard/src/components/revision/RevisionCard.stories.tsx [hunk 3]
```diff
@@ -54,8 +54,9 @@ Default.args = {
   revisionDate: new Date('2021-01-01 00:00:00'),
   previousVersion: '1.12.31',
   currentVersion: '1.12.32',
-  onClick: () => {},
-  onRefreshClick: () => {},
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  // onClick: () => {},
+  // onRefreshClick:() => {},
```
Comment (hdm23061993) [inline dashboard/src/components/revision/RevisionCard.stories.tsx:59]: Not needed. Please remove them

Comment (hdm23061993) [inline dashboard/src/components/revision/RevisionCard.stories.tsx:59]: Not needed. Please remove them

**Comment by hdm23061993** [comment]
Please use a logical title for your PR
Also, please add details for the changes made in the PR description

**Comment by MosheOgalbo** [comment]
I fixed it

**Comment by harshit-mehtaa** [comment]
Please fix PR title and description

**Pull request closed without merging**